src/users/users.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { User } from './user.entity';
+import { UsersController } from './users.controller';
+import { UsersService } from './users.service';
+
+@Module({
+    imports: [TypeOrmModule.forFeature([User])],
```

Comment:
I think you can push your `app.module.ts` file that you've made with your Typeorm connection settings, don't worry I will deal with its conflict, just make sure all of your tests pass